course/templates/course/course_base.html

```diff
@@ -60,6 +60,21 @@ <h4>{% trans "Calendar" %}</h4>
         {% endcomment %}
     </div>
     <div class="col-sm-10" id="course-content">
+        {% if instance.is_past and instance.has_enrollment_closed %}
+            <div class="alert sticky-alert alert-default">
+                {% trans "This course has already ended. " %}
+                {% if later_instance %}
+                    <p>
+                        {% trans "The latest version of the course can be found at: " %}
+                        <a class="alert-link" href="{{ later_instance|url }}">
+                            <span lang="{{ later_instance.language }}">
+                                {{ later_instance.course.name|parse_localization }}: {{ later_instance.instance_name|parse_localization }}
```

Comment:
Hmmm. I'm not sure if colon is the best separator for combining name and instance name. However, it appears to be the one that is used in those few locations where these are combined. I used format `name (instance)` in MOOC-Jutut, but I'm not sure if it would be better or not.

If `parse_localization` would handle multiple substitution, then we could say something like `{{ later_instance|parse_localization }}`, which would use `CourseInstance.__str__` (which uses format `name: instance`). If we did something like that, then the format could be changed in a single location.

Reply:
I copied this format from the top nav, there the instance is listed as "course: instance". Though the thing that is missing here is the course code, but I thought that with that one it becomes a bit messy looking. The course code is already listed three times on the page as it is. 

![image](https://user-images.githubusercontent.com/43036333/87956804-21ea0700-cab8-11ea-8aba-e0c861dce65e.png)

Reply:
Hmm.. I think we should visit the format if we touch that menu. In addition, I guess Jutut should use the same format too...